security.rst

```diff
@@ -679,6 +679,21 @@ like this:
   this - this is useful when *whitelisting* URLs to guarantee access - some
   details are in :doc:`/security/access_control`.
 
+* ``IS_ANONYMOUS``: Only anonymous users are matched by this attribute.
+
+* ``IS_REMEMBERED``: Matches users authenticated using the 
+  :doc:`remember me functionality </security/remember_me>`, (i.e. a
+  remember-me cookie).
+
+* ``IS_IMPERSONATOR``: When the current session is an
```

Comment:
`the session is a user` sounds weird to me, I don't know it it's because I'm not native.
This is one tricky though, when I read this sentence it's like `is impersonator when is impersonated`...
What do you think of something like:
```rst
When the current user is  :doc:`impersonating <...>` another user.
```

Reply:
Yeah, this is a terrible one to describe. "When the current user is impersonating another user" is also 'wrong', because the current user is the impersonated one :)

Maybe?

```rst
When the user in this session is :ref:`impersonated <...>` by another user.
```

Reply:
You mean the user from a firewall point of view, but as the attribute I mean from a "real" point of view, and so this is confusing, because we are both "right". But I guess we should go for `When the current user is impersonated by another "real" user making the request`?

Reply:
It makes me think we should have thought about naming the attribute `IS_IMPERSONATED` :D